src/commands/prompt.ts

```diff
@@ -1,14 +1,52 @@
-import {logger} from "../logger";
-import {SlashCommandBuilder} from "@discordjs/builders";
-import {CommandInteraction, CacheType} from "discord.js";
-import {CommandType} from "../types";
+import { logger } from "../logger";
+import { inlineCode, SlashCommandBuilder, SlashCommandStringOption } from "@discordjs/builders";
+import { CommandInteraction, CacheType, TextChannel, MessageEmbed } from "discord.js";
+import { CommandType } from "../types";
+import { Config } from "../config";
 
 const promptModule: CommandType = {
-  data: new SlashCommandBuilder().setName("prompt").setDescription("Prompt?"),
+  data: new SlashCommandBuilder()
+    .setName("prompt")
+    .setDescription("Ask something?")
+    .addStringOption((opt: SlashCommandStringOption) =>
+      opt
+        .setName("question")
+        .setDescription("What is the question?")
+        .setRequired(true)
+    ),
   execute: async (interaction: CommandInteraction<CacheType>) => {
-    logger.info("hello from prompt");
-    await interaction.reply("hello from prompt");
+
+    let configChannel = Config?.prompt.channel
+    if (configChannel && configChannel?.length === 0) {
```

Comment:
id prefer the code follow this pattern
```js
if (condition) {
    do(thing);
    return;
}
do(something);
```
instead of 
```js
if(condition){
    do(thing);
} else {
    do(something);
}
```
I find it makes the code much more readable

Reply:
Facts, guard clauses FTW